include/InputManager.h

```diff
@@ -10,11 +10,13 @@ class InputManager
 	Camera* camera;
 	
 	InputManager(GLFWwindow *window, Camera* cam);
+	void ScrollWheel(double xoffset, double yoffset);
 	void CheckInput();
 	
 private:
 	float cursorX = 0;
 	float cursorY = 0;
+	bool shiftKey = 0;
```

Comment:
Why not set this to false as opposed to 0?

Reply:
I don't think it's a big deal. 0/1 or false/true,  this shouldn't be part of our coding standard.

Reply:
d